entries/QUnit.stack.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.stack">
+	<title>QUnit.stack()</title>
+	<signature>
+		<argument name="offset" type="Number" optional="true" default="2">
+			<desc>Set the stackTrace line offset.</desc>
+		</argument>
+	</signature>
+	<desc>Return a single line source reference from the stacktrace (call stack)</desc>
+	<longdesc>
+		<p>This method returns a single line string representing the stacktrace from where it was called.</p>
+		<p>If it's given a offset argument, QUnit.stack will return the correspondent line from the call stack.</p>
```

Comment:
This needs some more context, since the argument has a default, so there's always some slicing going on.